### PR TITLE
Ret 1966 check and submit skeleton pages restructure

### PR DESCRIPTION
**Pull request by JackReevies**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-1966

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "outDir": "./dist/",
     "module": "commonjs",
     "esModuleInterop": true,
+    "resolveJsonModule": true,
```
Comment (JackReevies) [inline tsconfig.json:6]: This allows us to import json files directly without having to do all the `readFileSync` stuff. This way gives us type checking with intellisense too

File: src/main/views/check-your-answers.njk [hunk 2]
```diff
@@ -92,28 +95,31 @@
...
           } 
         },
         {
           key: {
             text: personalDetails.title,
-            classes: "govuk-!-font-weight-regular"
+            classes: "govuk-!-font-weight-regular-m"
           },
           value: {
+             text: "Not Selected" if userCase.preferredTitle
```
Comment (marcjudge) [inline src/main/views/check-your-answers.njk:120]: Logic should be reversed

File: src/main/views/check-your-answers.njk [hunk 3]
```diff
@@ -170,194 +165,311 @@
...
             ]
           } 
+        },
+        {
+          key: {
+            text: personalDetails.howToBeContacted,
+            classes: "govuk-!-font-weight-regular-m"
+          },
+          value: {
+            text: userCase.updatePreference
```
Comment (marcjudge) [inline src/main/views/check-your-answers.njk:175]: claimant_contact_preference is the correct property name I believe

File: src/main/views/check-your-answers.njk [hunk 4]
```diff
@@ -170,194 +165,311 @@
...
+  {% endif %}
+  
+  {% for respondent in respondents %}
   {{ 
     govukSummaryList({
       rows: [
         {
           key: {
-            text: yourPreferences.header,
+            text: respondentDetails.header,
```
Comment (marcjudge) [inline src/main/views/check-your-answers.njk:239]: This should be dynamic "respondent "+ respondent.respondentNumber

**Comment by Itay-Kirsh** [comment]
The updated fields for sex/gender related questions are:
claimantSex, claimantGenderIdentitySame, claimantGenderIdentity, preferredTitle, otherTitlePreference

File: src/main/definitions/case.ts [hunk 5]
```diff
@@ -40,6 +40,8 @@ export interface Case {
   claimantRepresentedQuestion?: YesOrNo;
   caseType?: CaseType;
   caseTypeId?: CaseTypeId;
+  gender?: string;
```
Comment (marcjudge) [inline src/main/definitions/case.ts:43]: I don't think we need this property. claimantSex should be the value we reference

Comment (marcjudge) [reply src/main/definitions/case.ts:43]: @olyode

Comment (olyode) [reply src/main/definitions/case.ts]: Yes referencing claimantSex, gender is now removed

**Review by marcjudge — APPROVED** [review]

File: src/main/resources/locales/en/translation/check-your-answers.json [hunk 6]
```diff
@@ -32,32 +32,39 @@
...
   },
   "employmentDetails": {
     "header": "Employment details",
+    "didYouWorkFor": "Did you work for the organisation or person you're making your claim against?",
+    "isStillWorking": "Are you still working for the organisation or person you're making your claim against?",
     "jobTitle": "Job title",
-    "startDate": "Start date",
-    "noticePeriod": "Notice period",
+    "startDate": "Employment start date",
+    "noticePeriod": "Do you have a written contrat with notice period?",
```
Comment (lvo700) [inline src/main/resources/locales/en/translation/check-your-answers.json:47]: Typos lines 43, 48. Spelling line 76 'discrimination'

Comment (olyode) [reply src/main/resources/locales/en/translation/check-your-answers.json]: Nice catch, resolved

**Review by lvo700 — APPROVED** [review]

**Review by JustMehmet — DISMISSED** [review]

**Pull request closed without merging**